sdk/mediaservices/Microsoft.Azure.Media.LiveVideoAnalytics.Edge/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-preview (2020-06-11)
```

Comment:
This will need to match the version in the project so you need to update it to `1.0.0-preview.1`